app/uk/gov/hmrc/pensionschemereturn/transformations/nonsipp/MemberPaymentsTransformer.scala

```diff
@@ -44,7 +45,12 @@ class MemberPaymentsTransformer @Inject()(
       schemeMadeTransferOut = memberPayments.memberDetails.exists(_.transfersOut.nonEmpty),
       lumpSumReceived = memberPayments.lumpSumReceived,
       pensionReceived = false,
-      surrenderMade = false,
+      surrenderMade = memberPayments.benefitsSurrenderedDetails match {
+        case SectionDetails(made @ true, completed @ true) => true
```

Comment:
can we refactor here like below if don't use the made and the completed fields ?

`surrenderMade = memberPayments.benefitsSurrenderedDetails match {
        case SectionDetails(true, true) => true
        case SectionDetails(true, false) => false
        case SectionDetails(false, true) => false
        case SectionDetails(false, false) => false // shouldn't happen but adding just in case
      }`

Reply:
it's just so when devs are looking at the code the can figure out what they mean rather than seeing true, false